.gitignore

```diff
@@ -19,8 +19,7 @@ logcat-*.txt
 apk-sizes-*.txt
 *.rawproto
 *.binlog
-src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props
```

Comment:
Was this removal intentional? This file is generated?

Reply:
There is a `src/Xamarin.Android.Build.Tasks/.gitignore`, where we actually had some things duplicated.

I moved the rules that seemed like they should be moved to `src/Xamarin.Android.Build.Tasks/.gitignore`.